app/core/urls.py

```diff
@@ -16,10 +16,11 @@
         r'^i18n/',
         include('django.conf.urls.i18n')
     ),
-    url(r'^candidates/$',
-        views.candidates_view,
-        name='candidates'
-    ),
+    # activate this url on the due date
```

Comment:
Why remove this? Should be enough to just not show the menu

Reply:
The url is enabled again.